terraform/modules/cluster/eks.tf

```diff
@@ -44,122 +58,77 @@ module "eks_blueprints" {
       type        = "ingress"
       self        = true
     }
-    # Recommended outbound traffic for Node groups
-    egress_all = {
-      description      = "Node all egress"
-      protocol         = "-1"
-      from_port        = 0
-      to_port          = 0
-      type             = "egress"
-      cidr_blocks      = ["0.0.0.0/0"]
-      ipv6_cidr_blocks = ["::/0"]
-    }
-
-    # Allows Control Plane Nodes to talk to Worker nodes on Karpenter ports.
-    # This can be extended further to specific port based on the requirement for others Add-on e.g., metrics-server 4443, spark-operator 8080, etc.
-    # Change this according to your security requirements if needed
-    ingress_nodes_karpenter_port = {
-      description                   = "Cluster API to Nodegroup for Karpenter"
-      protocol                      = "tcp"
-      from_port                     = 8443
-      to_port                       = 8443
-      type                          = "ingress"
-      source_cluster_security_group = true
-    }
-
-    ingress_nodes_load_balancer_controller_port = {
-      description                   = "Cluster API to Nodegroup for Load Balancer Controller"
-      protocol                      = "tcp"
-      from_port                     = 9443
-      to_port                       = 9443
-      type                          = "ingress"
-      source_cluster_security_group = true
-    }
-
-    ingress_nodes_metric_server_port = {
-      description                   = "Cluster API to Nodegroup for Metric Server"
-      protocol                      = "tcp"
-      from_port                     = 4443
-      to_port                       = 4443
-      type                          = "ingress"
-      source_cluster_security_group = true
-    }
   }
 
   # Add karpenter.sh/discovery tag so that we can use this as securityGroupSelector in karpenter provisioner
   node_security_group_tags = {
     "karpenter.sh/discovery" = var.environment_name
   }
 
-  managed_node_groups = {
-    mg_5 = {
-      node_group_name = "managed-ondemand"
-      instance_types  = ["m5.large"]
-      subnet_ids      = local.private_subnet_ids
-      min_size        = local.default_mng_min
-      max_size        = local.default_mng_max
-      desired_size    = local.default_mng_size
+  eks_managed_node_groups = {
+    managed-ondemand = {
+      instance_types = ["m5.large"]
 
-      ami_type        = "AL2_x86_64"
```

Comment:
The way EKS Blueprints managed this is a bit different from the EKS module. By default the `ami_type` is `AL2_x86_64`, and the release version is selected at the time of creation. I'm not sure what the explicit release version is used for in this context, typically we pull the latest version which includes the latest patches/etc. at the time of nodegroup creation. Let me know if this release version need to be put back

Ref: https://github.com/aws/containers-roadmap/issues/1945